### ISSUE TITLE
Possible side effect of the new CLI option for external JSON config file

### ISSUE THREAD

**Issue by noeldelgado** [jkphl/svg-sprite#173]
Hi Joschi, since today (version `1.3.2`) I am experiencing an error when running the following cmd:

```sh
> svg-sprite -s --symbol-dest=svg --symbol-prefix=.svg- --ss=sprite.svg --si --sx --shape-id-generator=svg-%s --dest=views/partials public/svg/*.svg
```

Output:

```sh
.../node_modules/svg-sprite/bin/svg-sprite.js:213
	if (!argv[mode] && !(mode in JSONConfig.mode)) {
	                                       ^
	                                       
TypeError: Cannot use 'in' operator to search for 'css' in undefined
    at Object.<anonymous> (.../node_modules/svg-sprite/bin/svg-sprite.js:213:41)
    at Array.forEach (native)
    at Object.<anonymous> (.../node_modules/svg-sprite/bin/svg-sprite.js:212:44)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:457:10)
    at startup (node.js:136:18)
    at node.js:972:3
```

Targeting version `1.3.1` will work as expected.

Thanks!

**Comment by jkphl** [comment]
@noeldelgado Yeah, unfortunately looks like a regression I didn't capture. Sorry for that! I'll take care of it tomorrow!

**Comment by noeldelgado** [comment]
Fixed on [1.3.3](https://github.com/jkphl/svg-sprite/blob/master/CHANGELOG.md#133-bugfix-release-2016-04-28).
